src/dlgTriggerEditor.cpp

```diff
@@ -8823,7 +8854,11 @@ void dlgTriggerEditor::slot_editorContextMenu()
 
 QString dlgTriggerEditor::generateButtonStyleSheet(const QColor& color, const bool isEnabled)
 {
-    if (color.isValid()) {
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
```

Comment:
Hang on - you still need the check for a valid colour here as well as it not being the transparent one don't you?